package.json

```diff
@@ -23,6 +23,8 @@
         "kafka"
```

Comment:
please add links from kafka.producers.fakerjs.* settings to the producing doc

Reply:
fixed for all settings. The description is not very helpful for the moment, but in the future, we could improve it with screenshots to see in action the settings result.